WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/PrefMasterSwitchToolbarView.kt

```diff
@@ -0,0 +1,222 @@
+package org.wordpress.android.ui.prefs.notifications
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.View.OnClickListener
+import android.view.View.OnLongClickListener
+import android.widget.CompoundButton
+import android.widget.CompoundButton.OnCheckedChangeListener
+import android.widget.LinearLayout
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.widget.SwitchCompat
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.ViewCompat
+import org.wordpress.android.R
+import org.wordpress.android.ui.prefs.AppPrefs
+import org.wordpress.android.util.redirectContextClickToLongPressListener
+
+/**
+ * Custom view for master switch in toolbar for preferences.
+ */
+class PrefMasterSwitchToolbarView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0,
+    defStyleRes: Int = 0
+) : LinearLayout(context, attrs, defStyle, defStyleRes),
+        OnCheckedChangeListener,
+        OnLongClickListener,
+        OnClickListener {
+    private var mHintOn: String? = null
```

Comment:
We usually don't use Hungarian notation in Kotlin classes. Just in Java for consistency 👍